### ISSUE TITLE
[3.1.0] One at a time printing pause on last printed part

### ISSUE THREAD

**Issue by Witgang** [Ultimaker/Cura#3071]
slight ordering bug in g code when printing one at a time
`G0 F7200 X37.632 Y89.646

G1 F1800 X36.739 Y90.54 E2673.73615

G1 X36.633 Y90.646

;TIME_ELAPSED:1234.398597

G1 F1500 E2667.23615

M107

M190 S60

G0 F7200 X36.633 Y90.646 Z19.35

G0 X147.803 Y66.716

;LAYER_COUNT:21

;LAYER:0`

**Application Version**
3.1.0

**Platform**
Windows 10

**Steps to Reproduce**
just throw 2 small objects in and move them away from each other then select "One at a time" and  let it slice then save file.

**Actual Results**
when printing the print head pauses on the last layer of the first printed part then it raises and moves on to print the next part.
  the problem with this is that it leaves a small blob or sometimes even melting the part ruining the finish.

**Expected results**
the print head should raise before waiting for heated bed to reach temp

**Additional Information**
so the  "M190 S60" should be generated after "G0 F7200 X36.633 Y90.646 Z19.35"

**Comment by Witgang** [comment]
also(unrelate to the issue) please for god sake increase the auto slicing time, i'm to used to it to turn it off but it gets me every single time i have 2 or more objects can't move one and then the other, i have to wait for slicing between each item move and with 10 or so it's reaaaly painful and is a chore to go disable it then remember to enable it..

**Comment by ChrisTerBeke** [comment]
For the 2nd issue (which is gonna be hard to fix as it's related to how the entire slicing engine works), please file a new issue if you want. Keeping track of multiple issues in 1 thread is hard.

**Comment by Ghostkeeper** [comment]
I thought we fixed this here: https://github.com/Ultimaker/CuraEngine/commit/f32d2e3972cf9a17f921367a81868c605b22176a

But apparently not. I still see the following g-code with two small objects:

    ;TIME_ELAPSED:620.565057
    G1 F1500 E104.87266
    M107
    M190 S50 ; <- Waiting on temperature
    G0 F7200 X46.466 Y144.169 Z15 ; <- Move up a bit.

**Comment by Ghostkeeper** [comment]
It appears that heating up for the first layer temperature works properly, but not cooling down!

**Comment by Ghostkeeper** [comment]
Our project manager removed this from our planning. We'll keep it open bug we won't be getting around to debugging this soon.

**Comment by timangus** [comment]
That's... an interesting choice. If you're printing something intricate this can ruin the part, right at the last second!

**Comment by Nuttavoot** [comment]
This bug still exist.
